.github/workflows/node.js.yml

```diff
@@ -5,9 +5,11 @@ name: Node.js CI
 
 on:
   push:
-    branches: [ "master" ]
+    branches: 
+      - '*'
```

Comment:
It's unclear to me whether this is desirable on the main repo

Reply:
Reverted in 31fa026dc6b1e7e530b9fea7fef084789d887b57

Can mark as resolved.